scripts/get_weights.py

```diff
@@ -0,0 +1,85 @@
+import torch
+from safetensors.torch import save_file
+import urllib.request
+import os
+import sys
+import argparse
+
+data_path = os.path.join(os.path.dirname(__file__), "../data/")
+vocab_filename = "bpe_simple_vocab_16e6.txt"
+
+def ensure_data_dir(safetensors):
+    print("Ensuring empty data directory...")
+
+    if os.path.exists(data_path):
+        # Fail if conflicting files exist
+        files = os.listdir(data_path)
+        newfiles = [x for name in safetensors for x in (f"{name}.bin", f"{name}.safetensors")]
+        newfiles += [vocab_filename, f"{vocab_filename}.gz"]
+        conflicts = set(files) & set(newfiles)
+        if len(conflicts) != 0:
+            print("Error: please remove the following files from data directory:")
+            print(conflicts)
+            sys.exit()

```

Comment:
This should probably return an error code different from 0, this way the script can be composed with other scripts and subsequent part would not be run if this one fails.